besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogSubCommand.java

```diff
@@ -82,7 +83,14 @@ public void run() {
   }
 
   private static BesuController createBesuController() {
-    return parentCommand.besuCommand.buildController();
+    final DataStorageConfiguration config = parentCommand.besuCommand.getDataStorageConfiguration();
+    // disable limit trie logs to avoid preloading during subcommand execution
+    return parentCommand
```

Comment:
If it's easy enough to add a test for this that would be good

Reply:
@jframe I initially thought it was unfeasible as the initial configuration wasn't wired up, but on second look I spotted the configuration argument captors - thanks for the prompt!